pallets/asset/src/lib.rs

```diff
@@ -372,6 +331,15 @@ decl_module! {
         const AssetMetadataValueMaxLength: u32 = T::AssetMetadataValueMaxLength::get();
         const AssetMetadataTypeDefMaxLength: u32 = T::AssetMetadataTypeDefMaxLength::get();
 
+        // Remove all storage related to classic tickers in this module
+        fn on_runtime_upgrade() -> Weight {
+            use polymesh_primitives::storage_migrate_on;
+            storage_migrate_on!(StorageVersion, 1, {
```

Comment:
```suggestion
            storage_migrate_on!(StorageVersion, 2, {
```

Reply:
This version number here is the target version, which is version `2` now.  We removed the old migrations, which is why we don't have one for version `1` anymore.